.github/workflows/build-test-package.yml

```diff
@@ -7,6 +7,7 @@ jobs:
     uses: InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction/.github/workflows/build-test-cxx.yml@v5.4.0
     with:
       itk-cmake-options: '-DModule_MorphologicalContourInterpolation_BUILD_EXAMPLES:BOOL=ON'
+      itk-module-deps: 'ITKRLEImage@v1.0.1'
```

Comment:
Perhaps this is itk-rleimage?   Should itk-morphologicalcontourinterpolation also be a dependency?

Reply:
This should be right per: https://github.com/InsightSoftwareConsortium/ITKRemoteModuleBuildTestPackageAction?tab=readme-ov-file#build-test-cxx-overview